.travis.yml

```
@@ -60,7 +62,9 @@ before_script:
   - ./dev/travis/before_script.sh
 script:
   - if [ $TEST_SUITE == 'static' ]; then phpunit --configuration dev/tests/static/phpunit.xml.dist dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php; fi
-  - if [ $TEST_SUITE == 'unit' ]; then phpunit --configuration dev/tests/unit/phpunit.xml; fi
+  - if [ $TEST_SUITE == 'unit' ]; then phpunit --configuration dev/tests/unit/phpunit.xml --coverage-text --coverage-clover ./clover.xml; fi
```

It would be good to specify the correct path for the report straight away and the copy operation will not be needed
```suggestion
  - if [ $TEST_SUITE == 'unit' ]; then phpunit --configuration dev/tests/unit/phpunit.xml --coverage-text --coverage-clover ../ext/magento/adobe-stock-integration/clover.xml; fi
```

---

Do we need to specify path here at all if it's already set in phpunit.xml?